blog.md

```diff
@@ -83,19 +135,22 @@ ghost:
     S3_ACCESS_KEY: ${S3_ACCESS_KEY}
 ```
 
-As you can see by the image below ... which is hosted in the s3 bucket... things must be working.
+I then added the appropriate environment variables to my shell and ran `up` to spin up a local development environment of the application. Once it was running I was able to confirm that the plugin was working by uploading the following image to the s3 bucket via the Ghost image upload mechanism:
 
 ![](https://ci-demo-ghost-images.s3.amazonaws.com/2015/Nov/ghost_blog-1447652183265.png)
 
-## Continuous Integration / Deployment
-
 ### Pull Request
 
-Development workflow is `feature branch` based and `PRs` and `merges` to the git repo will kick of CI events.  Thus I'll want to create a feature branch of my changes and create a PR from that:
+All new work is done roughly fo in feature branches and Pull Requests are made to  the `development` branch of the git repo which will Jenkins watches using the github pull request plugin (GHPR).  The development process looks a little something like this:
```

Comment:
You accidentally some words here. :smile: 
* « [...] done _roughly fo_ in feature [...] » 
* « [...] repo which _will_ Jenkins watches_ [...] »

You may wish to (surprise) break it up:
« All new work is done in feature branches. Pull requests are [...] »